src/HttpServer.php

```diff
@@ -222,13 +281,13 @@ function () use ($connection) {
     /**
      * Adds the body to the request before handling the request
      *
-     * @param string $body - body to be added to the request object
+     * @param StreamInterface $body - body to be added to the request object
      * @param ConnectionInterface $connection - client-server connection
      * @param Request $request - Adds the body to this request object
      */
-    public function sendBody($body, ConnectionInterface $connection, Request $request)
+    public function sendBody(StreamInterface $body, ConnectionInterface $connection, Request $request)
```

Comment:
Public? The method signature is a bit unclear to me. What does this do? Is this method even needed?